src/Raven.Server/Web/Studio/FolderPath.cs

```diff
@@ -110,6 +136,11 @@ private static string GetRestrictedFolder(bool isBackupFolder, RavenConfiguratio
 
     public class FolderPathOptions
     {
-        public List<string> List { get; } = new List<string>();
+        public List<string> List { get; private set; } = new List<string>();
```

Comment:
Maybe it should be sorted list, then we will not have to sort it later on?

Reply:
In windows, it is already sorted.

Reply:
Do we care that much? It is not a hot-path? How many items do we have there on average? 20?

Reply:
Ok, Changed to `SortedSet`.
SortedList uses 2 generic types